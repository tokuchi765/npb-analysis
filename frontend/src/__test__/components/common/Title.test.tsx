import React from 'react';
import Enzyme from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Title from '../../../components/common/Title';

Enzyme.configure({ adapter: new Adapter() });

describe('Titleコンポーネントテスト', () => {
  it('スナップショット作成', () => {
    const tree = renderer.create(<Title children={'タイトル'} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
