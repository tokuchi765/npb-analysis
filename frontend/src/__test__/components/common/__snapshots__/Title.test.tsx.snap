// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titleコンポーネントテスト スナップショット作成 1`] = `
<h2
  className="MuiTypography-root MuiTypography-h6 MuiTypography-colorPrimary MuiTypography-gutterBottom"
>
  タイトル
</h2>
`;
